plugins/clusternet.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: clusternet
+spec:
+  homepage: https://github.com/clusternet/kubectl-clusternet
+  shortDescription: "Wrap multiple kubectl calls to Clusternet"
+  description: |
+    interacting with Clusternet.
+  version: "v0.2.0"
+  caveats: |
+    Usage:
+      $ kubectl clusternet
+
+    For additional options:
+      $ kubectl clusternet --help
+      or https://github.com/clusternet/kubectl-clusternet/blob/v0.2.0/doc/USAGE.md
```

Comment:
Please use a stable URL here, for example by pointing to HEAD. I understand your motivation to point to the exact docs for this release, however it does not play well with our automated approver bot (which does not like when text fields change).